src/components/SimpleTable.vue

```diff
@@ -0,0 +1,209 @@
+<template>
+  <section class="aph-simple-table">
+    <table class="table-wrapper">
+      <thead>
+        <tr class="table-header-row">
+          <th v-for="key in columns"
+            @click="sortBy(key)"
+            class="header-cell"
+            :class="{ active: sortKey === key }">
+            {{ key | capitalize }}
+            <span class="arrow" :class="sortOrders[key] > 0 ? 'asc' : 'dsc'"></span>
+          </th>
+        </tr>
+      </thead>
+      <tbody class="table-body-wrapper">
+        <tr class="row" v-for="entry in filteredData">
+          <td class="cell" :class="[key, injectStyling(entry[key], entry, key)]" v-for="key in columns">
+            {{ formatEntry !== 'null' ? formatEntry(entry[key], entry, key) : entry[key] }}
+          </td>
+        </tr>
+      </tbody>
+    </table>
+  </section>
+</template>
+
+<script>
+
+export default {
+  props: {
+    data: {
+      default: [],
+      type: Array,
+    },
+    columns: {
+      default: [],
+      type: Array,
+    },
+    filterKey: {
+      default: '',
+      type: String,
+    },
+    formatEntry: {
+      default: null,
+      type: Function,
+    },
+    injectStyling: {
+      default: () => {
+      },
+      type: Function,
+    },
+  },
+
+  data() {
+    const sortOrders = {};
+    this.columns.forEach((key) => {
+      sortOrders[key] = 1;
+    });
+
+    return {
+      sortKey: '',
+      sortOrders,
+    };
+  },
+
+  computed: {
+    filteredData() {
+      const sortKey = this.sortKey;
+      const filterKey = this.filterKey && this.filterKey.toLowerCase();
+      const order = this.sortOrders[sortKey] || 1;
+      let data = this.data;
+      if (filterKey) {
+        data = data.filter((row) => {
```

Comment:
Could probably clean this up using `_.chain`

Reply:
I'll see if I can implement this.

Reply:
This is what I got to: 

data = _
          .chain(data)
          .filter((row) => {
            return Object.keys(row).some((key) => {
              return String(row[key]).toLowerCase().indexOf(filterKey) > -1;
            });
          })
          .value();

Not sure if that's much better or if I'm using it right. I think I'm probably gonna keep it the way I had it, cause I'm not sure if the code I came up with is any kind of improvement.